dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/DataSourceUser.java

```diff
@@ -58,4 +58,64 @@ public class DatasourceUser {
      * update time
      */
     private Date updateTime;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
+        this.userId = userId;
+    }
+
+    public int getDataSourceId() {
+        return datasourceId;
+    }
+
+    public void setDataSourceId(int datasourceId) {
+        this.datasourceId = datasourceId;
+    }
+
+    public int getPerm() {
+        return perm;
+    }
+
+    public void setPerm(int perm) {
+        this.perm = perm;
+    }
+
+    public Date getCreateTime() {
+        return createTime;
```

Comment:
You don't need to add `get`, `set` and `toString` method.

Reply:
OK